### PR TITLE
Describe multi-instance completion condition

### LINKED ISSUE #508: Describe multi-instance completion condition
Zeebe may soon gain support for multi-instance completion condition, see https://github.com/camunda-cloud/zeebe/issues/5439 and https://github.com/camunda-cloud/zeebe/pull/8321.

This new feature should be documented [here](https://docs.camunda.io/docs/reference/bpmn-processes/multi-instance/multi-instance/). A new paragraph about the completion condition should be added describing how it can be set and what behavior it specifies. In addition, its usage should be added to the [xml example](https://docs.camunda.io/docs/reference/bpmn-processes/multi-instance/multi-instance/#xml-representation).

### PR DESCRIPTION
**Pull request by Lzgabel**
This adds a description of how completionCondition can be defined for multi-instance.

closes  #508

### CODE REVIEW

File: docs/reference/bpmn-processes/multi-instance/multi-instance.md
Comment (korthout) [inline docs/reference/bpmn-processes/multi-instance/multi-instance.md:119]: ❌  Let's improve this example. 

We could use the `result` or `results` variables in an expression, e.g. `= result.isSuccessful` or `= count(results[item = true]) > 3` (at least 3 times the value `true` in the results list). We could also just use the moment of completion, something like: `= list contains([6,7], today().weekday)` to check that it's the weekend.

File: docs/reference/bpmn-processes/multi-instance/multi-instance.md
Comment (korthout) [inline docs/reference/bpmn-processes/multi-instance/multi-instance.md:100]: 🔧 I'd prefer to take the exclusive gateway conditions as example and describe the different scenarios in text, rather than in a list of options. That can makes this description less pseudo-code and more descriptive of its function.

**Review by korthout — CHANGES_REQUESTED** [review]
Thanks @Lzgabel for these docs 🙇 . I see a couple things we can improve.

Generally, I think we should take the [exclusive gateway conditions](https://docs.camunda.io/docs/reference/bpmn-processes/exclusive-gateways/exclusive-gateways/#conditions) as an example for this description.

Please have a look

**Comment by korthout** [comment]
Even though this feature is part of release 1.3.0 I've removed the `release/1.3.0` label, because we want to wait with publishing these docs until the modeler also offers support for this feature. This can be tracked [here](https://github.com/camunda/camunda-modeler/issues/2641).

File: docs/reference/bpmn-processes/multi-instance/multi-instance.md
Comment (korthout) [inline docs/reference/bpmn-processes/multi-instance/multi-instance.md:105]: 🙃  Tiny nitpick
```suggestion
= count(["a", "b", "c", "d"]) > 3
```

**Review by korthout — COMMENTED** [review]
👍  This looks great, thank @Lzgabel.

@christinaausley please have a look at this as well.

I'll hold off with the approval, because of the Modeler dependency. We can merge this after https://github.com/camunda/camunda-modeler/issues/2641 is merged.

**Comment by christinaausley** [comment]
From a technical writing perspective, this looks good to go.

**Review by korthout — APPROVED** [review]
With the Modeler's pre-release of `5.0.0-alpha.0` ([link](https://github.com/camunda/camunda-modeler/releases/tag/v5.0.0-alpha.0)), I think this is ready to be merged.

@Lzgabel, it seems there is a conflict. This is because the bpmn docs (incl. the multi-instance) have moved from the reference to the modeler. You can now find it here: https://docs.camunda.io/docs/components/modeler/bpmn/multi-instance/multi-instance/. Please resolve the conflict, and then we can merge this 🙇

**Comment by Lzgabel** [comment]
Hi @korthout. Sorry for the late handling of the conflict, we are celebrating Chinese New Year recently. 🙇  

Happy lunar new year!

**Comment by korthout** [comment]
Hi @Lzgabel Oops, sorry for disturbing your celebrations 🙇 Thanks for fixing it, nonetheless ❤️ 

Happy Lunar New Year to you too! 🎉

**Review by korthout — APPROVED** [review]
LGTM 🏅

**Pull request merged**
